unit-test.md

```diff
@@ -0,0 +1,19 @@
+Running unit tests
+==============
+
+To run the  unit tests, run the `run-tests.sh` script.  You will need to set these environment variables:
+
+- `SPARK_HOME`: your local copy of Apache Spark. Look at download_spark_dependencies.sh for more details.
+- `HYPEROPT_FMIN_SEED`: the random seed. You need to get its value from `.travis.yml`.
+
+For example:
+
+```bash
+hyperopt$ HYPEROPT_FMIN_SEED=3 SPARK_HOME=/usr/local/lib/spark-2.4.0-bin-hadoop2.7 ./run_tests.sh
```

Comment:
Do you have explicit recommendations which Spark version to use? If so, we should document that.

Reply:
I'd recommend we change this to 2.4.3 (latest) and in our documentation, state which versions are tested by the CI.  If users think it's needed, we could test multiple versions.